src/api.coffee

```diff
@@ -152,10 +156,18 @@ start = (bootstrapData) ->
   apiHelper TaskPlanStatsActions, TaskPlanStatsActions.load , TaskPlanStatsActions.loaded, 'GET', (id) ->
     url: "/api/plans/#{id}/stats"
 
+  # Note: the below exercise endpoints share the same store.
+  # The contents of the json payload is identical, except the second includes an is_excluded flag
+  # since it operates at the course level
+  #
+  # This one loads using an ecosystemId
   apiHelper ExerciseActions, ExerciseActions.load,
     ExerciseActions.loaded, 'GET', (ecosystemId, pageIds, requestType = 'homework_core') ->
-      page_id_str = pageIds.join('&page_ids[]=')
-      url: "/api/ecosystems/#{ecosystemId}/exercises/#{requestType}?page_ids[]=#{page_id_str}"
+      url: "/api/ecosystems/#{ecosystemId}/exercises/#{requestType}?#{arrayToParams(pageIds, 'page_ids')}"
+  # And this one loads using a courseId
+  apiHelper ExerciseActions, ExerciseActions.loadForCourse,
+    ExerciseActions.loadedForCourse, 'GET', (courseId, pageIds, requestType = 'homework_core') ->
+      url: "/api/courses/#{courseId}/exercises/#{requestType}?#{arrayToParams(pageIds, 'page_ids')}"
 
```

Comment:
This is loading exercises from two different endpoints, but saving them into the same store. I think that's ok because the store only stores them by the pageIds and doesn't actually have any course/ecosystem logic.  If it does turn out to later,  then we should split them apart.